playbooks/lae.yml

```diff
@@ -0,0 +1,22 @@
+---
+# by default this playbook runs in the staging environment
+# to run in production, pass '-e runtime_env=production'
+- name: build the Latin American Ephemera site
+  hosts: lae_{{ runtime_env | default('staging') }}
+  remote_user: pulsys
+  become: true
+
+  vars_files:
+    - ../group_vars/lae/{{ runtime_env | default('staging') }}.yml
```

Comment:
I like that the default is staging, and that you include in the top how to run production